src/functions.pl

```diff
@@ -910,17 +910,17 @@ sub mainLoop_initialized {
 		$pos = " : $char->{pos_to}{x},$char->{pos_to}{y} " . $field->name if ($char->{pos_to} && $field);
 		my $aiSeq = join(",", @ai_seq);
 		# Translation Comment: Interface Title with character status
-		$title = TF("%s B%s (%s), J%s (%s) : w%s%s [%s] - %s",
-			$charName, $char->{lv}, $basePercent . '%',
+		$title = TF("[%s] %s B%s (%s), J%s (%s) : w%s%s [%s] - %s",
```

Comment:
maybe it will be better?
```perl
$title = TF("%s B%s (%s), J%s (%s) : w%s%s [%s] - %s",
$current_profile_name ? '[$current_profile_name] $charName': $charName, $char->{lv}, $basePercent . '%',			$char->{lv_job}, $jobPercent . '%',
```